examples/Atlas/test/runAtlasWalkingNarrowStairs.m

```diff
@@ -45,7 +45,7 @@ function runAtlasWalkingNarrowStairs()
 nq = r.getNumPositions();
 
 % Generate boxes and IRIS safe terrain regions
-l = 0.22;
+l = 0.26;
```

Comment:
stairs were made slightly longer to accommodate 2/3 of the foot, which more accurately reflects what we can do in hardware